src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/Sidebar/LearnMore/contents.js

```diff
@@ -25,7 +21,7 @@ const getContents = ( techSupportLink, fullServiceCreativeTeamLink ) => {
 				{
 					heading: __( 'Why we ask', 'wp-module-onboarding' ),
 					description: __(
-						`We use this to help offer the best WordPress setup, features and suggestions for your site.`,
+						`We want to help everyone get the most out of their WordPress site and this setup, so we use this to help optimize for a great experience and periodically when we offer tips, reminders and recommendations.`,
```

Comment:
Hey David!
Is the WordPress `site` used intentionally? Or should translations be used?

Reply:
Yes lets substitute `site` -- good catch!